bec_widgets/applications/views/device_manager_view/device_manager_dialogs/config_choice_dialog.py

```diff
@@ -0,0 +1,48 @@
+from qtpy.QtWidgets import QDialog, QHBoxLayout, QLabel, QPushButton, QSizePolicy, QVBoxLayout
+
+
```

Comment:
There is no need to ask the user whether one wants to discard the config if the config is empty